api/README.md

```diff
@@ -12,3 +12,110 @@
   * inversify.ts, IoC configuration
   * inversify.test.ts, IoC configuration for test
   * server.ts, Hapi Server configuration
+
+## Auth0 Integration
+
+### Configuring Auth0
```

Comment:
Should these steps just be scripted, and require some param passing?

Reply:
yea I would say so